Other options are ignored when using `--stdin`

For example:
```
# echo "let foo; console.log(foo);" | xo --stdin --esnext

<text>
  1:2  error  Unexpected token let
```
Looks like the cli isn't passing the options to `lintText()`:
https://github.com/sindresorhus/xo/blob/master/cli.js#L74

--- a/cli.js
+++ b/cli.js
@@ -71,7 +71,7 @@ if (input[0] === '-') {
 
 if (opts.stdin) {
 	getStdin().then(function (str) {
-		log(xo.lintText(str));
+		log(xo.lintText(str, opts));
 	});
 
 	return;

